plugins/kafka/include/kafka/operator.hpp

```diff
@@ -36,9 +36,6 @@ namespace tenzir::plugins::kafka {
 
 namespace {
 
-// Default topic if the user doesn't provide one.
-inline constexpr auto default_topic = "tenzir";
-
```

Comment:
We decided to remove the default topic. Should this be mentioned in the changelog?

Reply:
Yes, I think so. I'd call it a bug fix, as previously we did not properly respect Kafka's default topic because of this.